src/raster/r.droka/r.droka.py

```diff
@@ -219,6 +220,7 @@ def main():
         y = float(point.split("|")[1])
         # print x,y,z
         # Calcolo cost (sostituire i punti di partenza in start_raster al pusto di punto)
+        # (Cost calculation (replace the starting points in start_raster at the point of the point))
```

Comment:
Can an Italian-speaking contributor approve this translation? I couldn't understand the original sentence using translation tools and the context.